excretor/src/tummy.rs

```diff
@@ -123,4 +123,18 @@ impl Tummy {
             })
             .collect())
     }
+
+    pub async fn get_user_info(&self, user_id: &str) -> Result<User, sqlx::Error> {
+        let user = query_as!(
+            DBMessageAndUser,
```

Comment:
message is not necessary here. let's create another model

Reply:
pls check: [8d768fc](https://github.com/kossiitkgp/top-secret-stuff/pull/45/commits/8d768fc49827abe72eead526b03348ed604fa22a)